apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -121,6 +121,33 @@ const styles = {
   }
 };
 
+const collapserIconStyle = {
```

Comment:
NIT: again, for consistency, i would prefer to see `styles.collapserIcon.showHideButton`, `styles.audio.wrapper`, etc.. imo it also reads slightly better.